### PR TITLE
Windows installer: Use PowerShell to update PATH value to avoid 1024 char truncation

### LINKED ISSUE #2587: minikube not added to PATH: installer uses setx.exe which runcates PATHs longer than 1024 chars
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

Please provide the following details:

**Environment**:

**Minikube version** (use `minikube version`): v0.25.0
- **OS** (e.g. from /etc/os-release): Windows 10 Enterprise
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): hyperv
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): minikube-v0.25.1.iso
- **Install tools**: Windows Installer
- **Others**:
The above can be generated in one go with the following commands (can be copied and pasted directly into your terminal):
```
minikube version
echo "";
echo "OS:";
cat /etc/os-release
echo "";
echo "VM driver": 
grep DriverName ~/.minikube/machines/minikube/config.json
echo "";
echo "ISO version";
grep -i ISO ~/.minikube/machines/minikube/config.json
```

**What happened**: Installed minikube using the Windows Installer. The update_path.bat script used setx.exe to add minikube's installation path to my system PATH variable. My system PATH is over 1024 chars long so setx.exe destructively truncated it. I had to restore my system PATH from registry backups.

**What you expected to happen**: minikube is installed without destructively changing my system PATH.

**How to reproduce it** (as minimally and precisely as possible): Start with a system PATH environment variable > 1024 chars. Run the minikube Windows installer.

**Output of `minikube logs` (if applicable)**: N/A

**Anything else do we need to know**: N/A

### PR DESCRIPTION
**Pull request by blueelvis**
1. Fixes the issue where setx was truncating PATH Environment variable on Windows when > 1024 characters
2. Made some Misc fixes to the NSIS Script so that the user is able to see the log during installation/uninstallation

Fixes #2587. 

Verified on the following -
1. Windows 10 Enterprise v1809
2. Windows 7 SP1 N Edition

### CODE REVIEW

**Comment by blueelvis** [comment]
I signed it

File: installers/windows/update_path.ps1 [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<#
+	.DESCRIPTION
```
Comment (tstromberg) [inline installers/windows/update_path.ps1]: I'm not familiar with this format -  but shouldn't .DESCRIPTION be indented the same amount as .PARAMETER?

**Comment by tstromberg** [comment]
@minikube-bot OK to test

File: installers/windows/update_path.ps1 (hunk 1 above)
Comment (blueelvis) [reply installers/windows/update_path.ps1]: It looks to be properly indented in my IDE -
image

**Comment by blueelvis** [comment]
Thanks! I do have the following questions though -

1. Is there any need for an installer which can do silent installations? Maybe we can then write some tests as well to check the PATH variable and ensure that `minikube version` returns proper output?
2. I downloaded the latest installer after this PR was merged but I didn't see this new code in the installer. Are releases not continuous for this?

**Comment by Al4DIN** [comment]
Hello I tried the same process to add Minikube to my path bu It's giving me this error.
I'm new to K8s so don't mind my small knowledge.

``` Exception :
System.ArgumentOutOfRangeException: La valeur doit être supérieure ou égale à zéro et inférieure à la taille de la mémoire tampon de la console dans cette dimension.
Nom du paramètre : top
La valeur réelle était -1.
   à System.Console.SetCursorPosition(Int32 left, Int32 top)
   à Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   à Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   à Microsoft.PowerShell.PSConsoleReadLine.Paste(Nullable`1 key, Object arg)
   à Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   à Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   à Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics) ```

**Pull request merged**